crates/vize_atelier_jsx/README.md

```
@@ -1,3 +1,7 @@
+<p align="center">
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)